python/vyos/vpp/nat/nat44.py

```diff
@@ -86,38 +86,34 @@ def delete_nat44_out_interface(self):
             is_add=False,
         )
 
-    # needs to check
     def add_nat44_interface_inside(self):
         """Add NAT44 interface"""
         self.vpp.api.nat44_interface_add_del_feature(
-            flags=1,
+            flags=0x20,
```

Comment:
This should be a constant. If there are issues with constants in the VPP module, we can just add it at the top of the script. Magic numbers are easy to forget, and we never know if VPP may change them.